rusk-wallet/src/gql.rs

```diff
@@ -20,6 +24,17 @@ pub struct GraphQL {
     status: fn(&str),
 }
 
+/// The tx_for_block returns a Vec<TransactionBlock> which contains
+/// the execution-core transaction, its id hash and gas spent
+pub struct TransactionBlock {
```

Comment:
It returns a tuple and if the type isnt defined concretely as a struct, the type checker doesnt like it and type inference seems to not work giving the "please define concrete type of hashmap" confusing errors 

The change is in scope of the PR because we use gql.rs to check if the endpoints are available in the `rues.rs`  fixing what was broken

Reply:
I understand this reasoning. If we introduce the struct, can we find a better name for it? A transaction-block associate that there are multiple transaction.